bears/python/BanditBear.py

```diff
@@ -21,21 +27,28 @@ class BanditBear:
     CAN_DETECT = {'Security'}
 
     @staticmethod
+    @deprecate_settings(bandit_skip_tests='bandit_skipped_tests')
     def create_arguments(filename, file, config_file,
-                         bandit_skipped_tests: typed_list(str) = (
+                         bandit_skip_tests: typed_list(str) = (
```

Comment:
"tests" is bandit terminology.
And it is confusing terminology.

We are using `foo_ignore` and `foo_select`